TelCo.ColorCoder/Colors.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Text;
+
+namespace TelCo.ColorCoder
+{
+
+    internal static class Colors
+    {
+        /// <summary>
+        /// Given a pair number function returns the major and minor colors in that order
+        /// </summary>
+        /// <param name="pairNumber">Pair number of the color to be fetched</param>
+        /// <param name="colorMapMajor"></param>
+        /// <param name="colorMapMinor"></param>
+        /// <returns></returns>
+        public static ColorPair GetColorFromPairNumber(int pairNumber, Color[] colorMapMajor, Color[] colorMapMinor)
+        {
+            // The function supports only 1 based index. Pair numbers valid are from 1 to 25
```

Comment:
This comment is an attribute of the parameter pairNumber. This is useful if it is part of  /// <param name="pairNumber">Pair number of the color to be fetched</param>.